build/charts/antrea/crds/packetcapture.yaml

```diff
@@ -73,6 +73,10 @@ spec:
                   message: "source.pod must be set when capturePoint is 'Source'"
                 - rule: "!(has(self.capturePoint) && self.capturePoint == 'Destination') || has(self.destination.pod)"
                   message: "destination.pod must be set when capturePoint is 'Destination'"
+                - rule: "!(has(self.packet) && self.packet.ipFamily == 'IPv4' && has(self.packet.protocol) && self.packet.protocol == 'ICMPv6')"
+                  message: "protocol cannot be 'ICMPv6' when ipFamily is 'IPv4'"
+                - rule: "!(has(self.packet) && self.packet.ipFamily == 'IPv6' && has(self.packet.protocol) && self.packet.protocol == 'ICMP')"
+                  message: "protocol cannot be 'ICMP' when ipFamily is 'IPv6'"
```

Comment:
I noticed that we don't even check for consistency between the `protocol` field and the transport header fields (`tcp`, `udp`, etc.), so maybe we should hold off on adding these rules after all? What do you think @hangyan

Reply:
> I noticed that we don't even check for consistency between the `protocol` field and the transport header fields (`tcp`, `udp`, etc.), so maybe we should hold off on adding these rules after all? What do you think @hangyan

I remember that the current behavior is that if the user misconfigures it, it will generate 'valid' BPF code and fail to capture any packets. It doesn’t seem like an error that would occur easily, so I think it’s fine not to add it.